redhat_access_insights/connection.py

```diff
@@ -169,6 +153,13 @@ def get_proxies(self, config):
                 conf_proxy = scheme + location
             logger.debug("CONF Proxy: %s", conf_proxy)
             proxies = {"https": conf_proxy}
+
+        if os.environ.get('NO_PROXY') and conf_proxy:
+            logger.debug("Invalid Proxy configuration. "
+                         "You have environment variable NO_PROXY set "
+                         "as well as 'proxy' set in your configuration file. "
+                         "This is invalid. Do not expect normal results.")
```

Comment:
I would  not say this configuration is invalid, unlesss we parse the NO_PROXY variable and found the insights base url in the PROXY list. Since we are not parsing NO_PROXY,  its enough to just say "NO_PROXY envronment variable will be ignored. "